src/core/helpers.mjs

```diff
@@ -182,3 +183,20 @@ export function validatePath (input) {
   }
   return input
 }
+
+/**
+ * Download logs files from all network pods and save to local solo log directory
+ * @param k8
+ *    an instance of core/K8
+ * @returns A promise that resolves when the logs are downloaded
+ */
+export async function getNodeLogs (k8) {
+  const pods = await k8.getPodsByLabel(['fullstack.hedera.com/type=network-node'])
+  for (const pod of pods) {
+    const podName = pod.metadata.name
+    const targetDir = `${SOLO_LOGS_DIR}/${podName}`
+    fs.mkdirSync(targetDir, { recursive: true })
+    await k8.copyFrom(podName, ROOT_CONTAINER, `${HEDERA_HAPI_PATH}/output/swirlds.log`, targetDir)
+    await k8.copyFrom(podName, ROOT_CONTAINER, `${HEDERA_HAPI_PATH}/output/hgcaa.log`, targetDir)
```

Comment:
it might be useful to put this into a try/catch block, log the error, then push it into an array.  after the for loop if there are any errors in the array throw an exception saying which nodes failed.

That way, if the first node fails, we could still get the logs for the other nodes which might be useful to diagnose the issue.

Reply:
Added try catch block
But did not throw exception so the test can continue to move on with deleting namespace / shutting down properly